deploy/LOCAL_DEVELOPMENT_INSTALL.md

```diff
@@ -0,0 +1,117 @@
+# seqr local development set up
+
+Instructions for setting up seqr on a local machine for development
+
+## Install dependencies
+
+Before installing, always check first to see if a dependency is already installed
+
+- [python 3](https://www.python.org/downloads/)
+- [gcloud](https://cloud.google.com/sdk/install)
+- [postgres](https://www.postgresql.org/download/)
+- [redis](https://redis.io/topics/quickstart)
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
```

Comment:
I'm actually currently using npm 6, it turns out. And it looks like npm 8 is actually the same as npm 7 but dropping support for node < 12, so that should also work as well (https://github.com/npm/rfcs/issues/445#issuecomment-945313849). In the interest of keeping this documentation from getting out of dat too quickly, I'm not going to specify a version here

Reply:
As a side note here, the versions of NPM/node that our docker images (these are from the Debian Bullseye packages) will use to build the assets are npm 7.5.2, and nodejs v12.22.5